tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -38,13 +41,7 @@ public async Task GivenUsersExist(Table table)
             var users = table.CreateSet<NewUserTable>();
             foreach (var user in users)
             {
-                var allOrganisations = _context.Get<IDictionary<string, Guid>>(ScenarioContextKeys.OrganisationMapDictionary);
-
-                var organisationId = Guid.Empty;
-                if (allOrganisations.ContainsKey(user.OrganisationName))
-                {
-                    organisationId = allOrganisations[user.OrganisationName];
-                }
+                var organisationId = GetOrganisationIdFromName(user.OrganisationName);
```

Comment:
Rather than returning a string from the 'GetOrganisationIdFromName' method, do you think it would be better to return the GUID? This way you don't need to parse it again seen as we have already got the GUID once.

Reply:
Thought about that alternative, but decided for returning string as -
this method is used in 3 places. 2/3 want to get the string representation of the GUID and I thought parsing in 1 place is better than .ToString()'ing in 2 places.

Reply:
ToString() is a lot less likely to fail than a Guid.Parse & it seems very inefficient to change it to a string to change it back to a Guid. Also I don't think you will need to call ToString in multiple places, String interpolation will do it for you. Please review.

Reply:
Oh, didn't realise that string interpolation does that. Well, that changes everything. I'll happily change the return type to Guid :]